src/engraving/rendering/dev/chordlayout.cpp

```diff
@@ -738,12 +738,13 @@ void ChordLayout::layoutArticulations(Chord* item, LayoutContext& ctx)
     if (keepArticsTogether) {
         // find out how many close-to-note artics there are, and whether there is a staff-anchored artic to align to
         for (Articulation* a : item->articulations()) {
-            if (a->layoutCloseToNote()) {
+            if (a->layoutCloseToNote() && a->visible()) {
```

Comment:
It might be clearer to do:
```c++
        for (Articulation* a : item->articulations()) {
            if (!a->visible()) {
                continue;
            }

            // here, the rest of the loop unmodified
```

Reply:
That's a cleaner way of writing it. Updated, thanks!